support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -254,18 +254,24 @@ object CreateZuoraSubscription {
 
 object DigitalSubscriptionGiftRedemption {
 
-  sealed trait SubscriptionState
+  sealed abstract class SubscriptionRedemptionState(val clientCode: String)
 
-  case class Unredeemed(subscriptionId: String) extends SubscriptionState
+  object Unredeemed {
+    val clientCode = "unredeemed"
```

Comment:
can this just be inlined like the others for simplicity?
If we change the value, presumably it would be relied on independently by the client, so we would want the test to check the actual value (or could just check the type is an `Unredeemed` if that is all we care about)

Reply:
It couldn't be accessed through `Unredeemed.clientCode` then in the way that we can access `Redeemed.clientCode` because `Unredeemed` is a case class not a case object